src/components/map/MapService.js

```diff
@@ -720,7 +720,10 @@ goog.require('ga_urlutils_service');
                 },
                 // Temporary until https://github.com/openlayers/ol3/pull/4964
                 // is merged upstream
-                cacheSize: 2048 * 3,
+                // Workaround: Set a cache size of zero when layer is
+                // timeEnabled see:
+                // https://github.com/geoadmin/mf-geoadmin3/issues/3491
+                cacheSize: layer.timeEnabled ? 0 : 2048 * 3,
```

Comment:
yes I wanted to ask you as I was unsure. We could maybe leave the 2048 default value for all others. I don't know about the side effects for sure.